modules/8_reinforcement_learning/README.md

```diff
@@ -0,0 +1,26 @@
+# Reinforcement Learning
+
+This module asks to solve one of `gym` environments.
+Check out `try_gym()` function to get perception of what `gym` is about.
+To complete this module you should:
+1. implement
+    i. `Agent`'s methods. Hyperparamthers `ALPHA`, `GAMMA` are initialized for you
+    ii. A part of `train()` function
+2. Add `q_values.npy` file saved by `agent.save(WEIGHTS_PATH)`
```

Comment:
Is there a visualization for Agent's task? To help students understand the goal

Reply:
There is `render()` in `try_gym()` to draw the environment. And the description in FrozenLakeEnv definition should also help.

Reply:
Great, thank you!